bundles/AdminBundle/Resources/public/js/pimcore/object/classes/class.js

```diff
@@ -1391,6 +1388,15 @@ pimcore.object.classes.klass = Class.create({
                     fieldValidation = node.data.editor.isValid();
                 }
 
+                if (fieldValidation && data.fieldtype === 'advancedManyToManyObjectRelation') {
```

Comment:
can we find a more generic implementation here so that  `data.fieldtype === 'advancedManyToManyObjectRelation'` is not necessary and `class.js`  does not need to know anything about the available types?